src/app/App.js

```diff
@@ -67,6 +69,14 @@ class App {
     }
 
     // ADD GETTERS BELOW
+        
```

Comment:
@junthehacker Sorry If I'm being picky, but we do have JSDocs here as well - eg https://github.com/CheaprEats/js-sdk/pull/23/files#diff-ac3c50ae15a631981374f09a64815339R81